src/test/java/hw7/dataproviders/TestDataProvider.java

```diff
@@ -0,0 +1,23 @@
+package hw7.dataproviders;
+
+import hw7.jdi.entities.TestData;
+import org.testng.annotations.DataProvider;
+
+import java.util.Map;
+
+public class TestDataProvider {
+
+    @DataProvider(name = "Data Sets Provider")
+    public Object[] getData() {
+        TestDataGetter testDataGetter = new TestDataGetter();
+        Map<String, TestData> testDataArray = testDataGetter.getTestData();
+        return new Object[]{
```

Comment:
This is completely prohibited to interact with data in this way. What should we do in case if data changed?

Reply:
Solved: I remove all these code lines from data provider method and reworked `getTestData`-method: now it's static and returns `Object[]`, not simple `Map`, as in last version